client/src/store/actions/mainActions.js

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable import/prefer-default-export */
+export const CHECKING_IF_AUTHED = 'CHECKING_IF_AUTHED';
```

Comment:
`mainActions` seems a bit grandiose ;)

Reply:
Hmm agreed, although it it is actions for the `main` component! Any other suggestions?

Reply:
Just a note, most of the other reducers don't have an actions page associated with them. I have a feeling what you've done here is better practice so we should discuss how we bring in the actions into the reducer to keep consistency throughout.

Reply:
Agreed, made issue #59 for this.